services/web/server/src/simcore_service_webserver/projects/projects_db.py

```diff
@@ -243,7 +300,11 @@ def engine(self) -> Engine:
             # FIXME: prefer None to raise an exception. Read https://stackoverflow.com/questions/1313812/raise-exception-vs-return-none-in-functions?answertab=votes#tab-top
             if not row:
                 raise ProjectNotFoundError(project_uuid)
-            return _convert_to_schema_names(row)
+
+            study = _convert_to_schema_names(row)
```

Comment:
``project``, not ``study`` (see comments above)

Reply:
you want me to refactor everything?